common/store.go

```diff
@@ -95,7 +96,18 @@ type EigenDAStore interface {
 	Verify(ctx context.Context, serializedCert []byte, payload []byte, opts CertVerificationOpts) error
 }
 
-// SecondaryStore is the interface for a key-value data store that uses keccak(value) as the key.
+// EigenDAV2Store is the interface for an EigenDA V2 data store as well as V2 memstore.
+type EigenDAV2Store interface {
+	Store
+	// Put inserts the given value into the key-value (serializedCert-payload) data store.
+	Put(ctx context.Context, payload []byte) (serializedCert []byte, err error)
+	// Get retrieves the given key if it's present in the key-value (serializedCert-payload) data store.
+	Get(ctx context.Context, versionedCert certs.VersionedCert) (payload []byte, err error)
+	// Verify verifies the given key-value pair.
+	Verify(ctx context.Context, versionedCert certs.VersionedCert, opts CertVerificationOpts) error
+}
+
+// PrecomputedKeyStore is the interface for a key-value data store that uses keccak(value) as the key.
```

Comment:
Looks like your merge reverted a change made by Sam. Interface is now called `SecondaryStore`, but you've modified the doc back to the old name `PrecomputedKeyStore`

Reply:
[fdd84a2](https://github.com/Layr-Labs/eigenda-proxy/pull/384/commits/fdd84a22e1cbd38cd0b82dc8675702e31e44bbdc)